cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -186,13 +209,17 @@ private AccessToken getNewAccessToken(ConnectionConfig connectionInfo, PrintStre
         credentialValue = reader.readLine(prompt);
       }
       properties.put(credential.getName(), credentialValue);
+      if (credential.getName().contains("username")) {
```

Comment:
the actual value was something like cdap.security.authentication.basic.username and I wasn't sure if this would change if the authentication method changed.